tox.ini

```diff
@@ -118,7 +118,7 @@ commands =
     !cov-!double: pytest --pyargs astropy {toxinidir}/docs {env:MPLFLAGS} {posargs}
     cov-!double: pytest --pyargs astropy {toxinidir}/docs {env:MPLFLAGS} --cov astropy --cov-config={toxinidir}/pyproject.toml --cov-report xml:{toxinidir}/coverage.xml {posargs}
 
-    double: python -c 'import sys; from astropy import test; test(); sys.exit(test())'
+    double: pytest --pyargs astropy
```

Comment:
This is not equivalent to before - we need to run the tests twice in the same go to make sure that no global state is changed. Maybe using something like pytest-repeat could help but we need to run all tests once and then run them all again.

Reply:
Urghh, I missed that second `test()`. Thanks!

Wouldn't something like
```
    double: pytest --pyargs astropy
    double: pytest --pyargs astropy
```

be fine in that case?

Reply:
No because the second call starts a new Python interpreter with a clean state.

Reply:
I think one would need something like `pytest --keep-duplicates astropy docs astropy docs`

Reply:
I thought it might not work with `--pyargs` but it seems this works too,
```
pytest --keep-duplicates --pyargs astropy astropy
```

Reply:
Oh nice find!